conf/helmfile.d/0600.prometheus-operator.yaml

```diff
@@ -15,450 +15,121 @@ releases:
 #   - https://github.com/coreos/prometheus-operator/tree/master/helm/prometheus-operator
 #   - https://github.com/coreos/prometheus-operator
 #
-- name: "prometheus-operator"
-  namespace: "monitoring"
+- name: prometheus-operator
+  namespace: monitoring
   labels:
-    chart: "prometheus-operator"
-    repo: "stable"
-    component: "monitoring"
-    namespace: "monitoring"
-    vendor: "coreos"
-    default: "true"
-  chart: "stable/prometheus-operator"
-  version: "5.19.0"
+    chart: prometheus-operator
+    repo: stable
+    component: monitoring
+    namespace: monitoring
+    vendor: coreos
+    default: true
+  chart: stable/prometheus-operator
+  version: 8.12.3
   wait: true
   values:
-    # Default values for prometheus-operator.
-    # This is a YAML-formatted file.
-    # Declare variables to be passed into your templates.
-
-    ## Provide a name in place of prometheus-operator for `app:` labels
-    ##
-    - nameOverride: ""
-
-      ## Provide a name to substitute for the full names of resources
-      ##
-      fullnameOverride: ""
-
-      ## Labels to apply to all resources
-      ##
-      commonLabels: {}
-      # scmhash: abc123
-      # myLabel: aakkmd
-
-      ## Create default rules for monitoring the cluster
```

Comment:
The big deletions in this file were badly needed! I'm a little surprised that we were able to delete the defaultRules section here, though. No strange behavior observed after this?

Reply:
The values were copied over from the helm chart, so anything that was removed is using the current defaults for the prometheus-operator version.

Having all the values caused some errors, as we upgraded the version of the operator, but kept the default values from a previous version.